tests/Lookup/Test_LookupNodeForTxBlock.cpp

```diff
@@ -122,11 +122,12 @@ BOOST_AUTO_TEST_CASE(testTxBlockStoring)
 
     std::pair<PrivKey, PubKey> pubKey1 = Schnorr::GetInstance().GenKeyPair();
 
-    TxBlock txblock(
-        TxBlockHeader(TXBLOCKTYPE::FINAL, BLOCKVERSION::VERSION1, 1, 1,
-                      BlockHash(), 0, get_time_as_int(), TxnHash(), StateHash(),
-                      StateHash(), 0, 5, pubKey1.second, 0, BlockHash()),
-        vector<bool>(1), vector<MicroBlockHashSet>(5), CoSignatures());
+    TxBlock txblock(TxBlockHeader(TXBLOCKTYPE::FINAL, BLOCKVERSION::VERSION1, 1,
+                                  1, BlockHash(), 0, get_time_as_int(),
+                                  TxnHash(), StateHash(), StateHash(), 0, 5,
+                                  pubKey1.second, 0, BlockHash()),
+                    vector<bool>(1), vector<MicroBlockHashSet>(5),
```

Comment:
Should use "true" instead of "1" into bool container to avoid ambiguous though they're identical in almost all compiler.